{"name": "test_reg_russian_username", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Incorrect username' in '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...\">powered by ТЕХНОАТОМ</p>\\n    <p>Unless explicitly silenced</p>\\n</div>\\n            </footer>\\n    \\n</body></html>'\n +  where '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...\">powered by ТЕХНОАТОМ</p>\\n    <p>Unless explicitly silenced</p>\\n</div>\\n            </footer>\\n    \\n</body></html>' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"f73b2bb0c7c1e173de6b7cf746e05592\")>.page_source\n +    where <selenium.webdriver.remote.webdriver.WebDriver (session=\"f73b2bb0c7c1e173de6b7cf746e05592\")> = <tests.test_ui.TestReg object at 0x116224650>.driver", "trace": "self = <tests.test_ui.TestReg object at 0x116224650>\n\n    def test_reg_russian_username(self):\n        \"\"\"\n        Тестируется возможность пройти регистрацию с русским значением поля username.\n        Приложение должно выдавать ошибку в силу следующих причин:\n        1. В любом сервисе данное поле заполняется на универсальном английском языке и почти всегда должно\n        быть уникально(это в данном случае предусмотрено)\n        2. Приложение подразумевает использование VK API, для работы которого необходимо передавать username на англ.\n        \"\"\"\n        username, email, password = generate_user(russian_username=True)\n        self.reg_page.reg(username, email, password)\n        self.builder.del_user(username)\n        allure.attach(self.driver.get_screenshot_as_png(),\n                      name='test_python_link',\n                      attachment_type=allure.attachment_type.PNG)\n>       assert \"Incorrect username\" in self.driver.page_source\nE       assert 'Incorrect username' in '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...\">powered by ТЕХНОАТОМ</p>\\n    <p>Unless explicitly silenced</p>\\n</div>\\n            </footer>\\n    \\n</body></html>'\nE        +  where '<html class=\"uk-notouch\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; ...\">powered by ТЕХНОАТОМ</p>\\n    <p>Unless explicitly silenced</p>\\n</div>\\n            </footer>\\n    \\n</body></html>' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"f73b2bb0c7c1e173de6b7cf746e05592\")>.page_source\nE        +    where <selenium.webdriver.remote.webdriver.WebDriver (session=\"f73b2bb0c7c1e173de6b7cf746e05592\")> = <tests.test_ui.TestReg object at 0x116224650>.driver\n\ntests/test_ui.py:91: AssertionError"}, "description": "\n        Тестируется возможность пройти регистрацию с русским значением поля username.\n        Приложение должно выдавать ошибку в силу следующих причин:\n        1. В любом сервисе данное поле заполняется на универсальном английском языке и почти всегда должно\n        быть уникально(это в данном случае предусмотрено)\n        2. Приложение подразумевает использование VK API, для работы которого необходимо передавать username на англ.\n        ", "attachments": [{"name": "test_python_link", "source": "a337aa4a-9bb1-41e3-a540-031d40976c1c-attachment.png", "type": "image/png"}], "start": 1590610496272, "stop": 1590610507077, "uuid": "f076f133-38b5-4001-8e4c-ab0cd7d14706", "historyId": "a80deb18a3fa2089d070cc670f7385a4", "testCaseId": "040c3145c8c4500e0bed14c3faa5a5f8", "fullName": "tests.test_ui.TestReg#test_reg_russian_username", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestReg"}, {"name": "host", "value": "MacBook-Air-Roman-2.local"}, {"name": "thread", "value": "20864-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}